src/routes/docs/products/auth/+layout.svelte

```diff
@@ -112,6 +116,10 @@
                 {
                     label: 'Multi-tenancy',
                     href: '/docs/products/auth/multi-tenancy'
+                },
+                {
+                    label: 'Storing user preferences',
```

Comment:
Let's rename this to "Preferences storage" to make it shorter and a noun to match others.